flink-core/src/test/java/org/apache/flink/metrics/reporter/JMXReporterTest.java

```diff
@@ -285,4 +289,23 @@ public long getMin() {
 			};
 		}
 	}
+
+	/**
+	 * Sets the JMX port range config key based on the fork number.
+	 * @param config Configuration instance to configure
+	 * @return Passed configuration instance
+	 */
+	private static Configuration setJmxPortRange(Configuration config) {
+		int forkNumber = 0;
+		try {
+			forkNumber = Integer.parseInt(System.getProperty("forkNumber"));
+		} catch (NumberFormatException ignored) {
+		}
+
+		int minPort = 9000 + forkNumber * 100;
```

Comment:
what does the forkNumber bring us here? wouldn't it be better to use it for the mapPort to increase the range of ports?